Source/src/WixSharp/Nsis/NsisBootstrapper.cs

```diff
@@ -115,7 +116,16 @@ public class NsisBootstrapper : NsisBootstrapperLegacy
         /// <summary>
         /// Gets or sets digital signature parameters for the bootstrapper.
         /// </summary>
-        public DigitalSignature DigitalSignature;
+        public DigitalSignature DigitalSignature { get; set; }
+
+        /// <summary>
+        /// Describes not supported windows versions.
+        /// <para></para>
+        /// If any version added, checks if current windows version is not supported.
+        /// <para></para>
+        /// If so - displays error and terminates installation (configurable via <see cref="NotSupportedWindowsVersions"/> properties).
+        /// </summary>
+        public NotSupportedWindowsVersions NotSupportedWindowsVersions { get; } = new NotSupportedWindowsVersions();
```

Comment:
I'd vote for something more shorter like
"MinVersions" or "MinReqWinVersions"

Reply:
It implies modification of internal logic. 
Current logic is exactly about restricting versions, but not allowing them.
I can suggest renaming it to RestrictedWinVersions, BannedWinVersions, ForbiddenWinVersions, if the length is the only consern.
@TigranAnchorfree, what do you think?

Reply:
Guys, since I have very little to input about NSIS I suggest you decide the best course of action and let me know when the PR is ready for merge

Reply:
I think the solution is probably more flexible it needs to be. An easier to option from discoverability point of view might be something like that 
```
bootstrapper.OS.MinVersion = WindowsVersion._7;
bootstrapper.OS.UnsupportedVersions = new[] { WindowsVersion._8 };`
```

Reply:
All good from my side , @oleg-shilo we can merge it , thank you Eugene !